project/settings.py

```diff
@@ -60,13 +60,13 @@
         CACHES = {
             'default': {
                 'BACKEND': 'django.core.cache.backends.memcached.MemcachedCache',
-                'LOCATION': '127.0.0.1:11211',
+                'LOCATION': os.environ.get('MAPIT_CACHE_LOCATION', '127.0.0.1:11211'),
```

Comment:
I feel like this env var should reference that it is for memcache, given that it can only be used for that (from the naming alone I'd expect it was more general like a file path). We already have a convention for the env var MEMCACHE_SERVERS (which is the env var [dalli can pick up automatically](https://github.com/petergoldstein/dalli/blob/c73e52bf2993877ad509938dd840f0544633e998/lib/dalli/client.rb#L35)) so if you use that it'd be consistent with other GOV.UK services.